### PR TITLE
Version2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/remove [hunk 1]
```diff
@@ -38,67 +30,15 @@ then
...
-ynh_remove_app_dependencies
-
-#=================================================
-# CLOSE A PORT
-#=================================================
-
-if yunohost firewall list | grep -q "\- 445$"
-then
-	ynh_script_progression --message="Closing port 445..."
-	ynh_exec_warn_less yunohost firewall disallow TCP 445
```
Comment (Salamandar) [inline scripts/remove:76]: You're not closing the port anymore?

File: scripts/restore [hunk 2]
```diff
@@ -6,62 +6,19 @@
...
-#=================================================
-ynh_script_progression --message="Reinstalling dependencies..." --weight=1
+ynh_restore_file --origin_path="$data_dir" --not_mandatory
 
-# Define and install dependencies
-ynh_install_app_dependencies $pkg_dependencies
+chown -R $app: "$data_dir"
 
 #=================================================
 # RESTORE VARIOUS FILES
```
Comment (Salamandar) [inline scripts/restore:31]: You should yunohost firewall allow on restore too

File: scripts/upgrade [hunk 3]
```diff
@@ -9,79 +9,21 @@
...
 # SPECIFIC UPGRADE
 #=================================================
 # UPDATE A CONFIG FILE
 #=================================================
 ynh_script_progression --message="Updating a configuration file..." --weight=1
 
-ynh_add_config --template="../conf/smb.conf" --destination="/etc/smb/smb.d/$app.conf"
-ynh_add_config --template="../conf/avahi-samba.service" --destination="/etc/avahi/services/$app.service"
+ynh_add_config --template="smb.conf" --destination="/etc/smb/smb.d/$app.conf"
+ynh_add_config --template="avahi-samba.service" --destination="/etc/avahi/services/$app.service"
```
Comment (Salamandar) [inline scripts/upgrade:27]: Same here, maybe run yunohost firewall allow

**Review by Salamandar — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**